docs/_pages/about.md

```diff
@@ -61,26 +61,11 @@ We heavily practice Test Driven Development and one of the promises TDD makes is
 So although you are free to go through the many examples here, please consider to analyze the many [unit tests](https://github.com/fluentassertions/fluentassertions/tree/master/Tests/Shared.Specs).
 
 ## Who is behind this project
-
-My name is [Dennis Doomen](https://twitter.com/ddoomen) and I work for Aviva Solutions (in The Netherlands). I maintain a [blog](http://www.continuousimprover.com/) on my everlasting quest for knowledge that significantly improves the way you build your key systems in an agile world. Fluent Assertions is one of those aspects of that.
-
-Notable contributors over the last year include.
-
-* [Adam Voss](https://github.com/adamvoss)
-* [Jonas Nyrup](https://github.com/jnyrup)
-* [Artur Krajweski](https://github.com/krajek)
-
-The [Fluent Assertions logo](./logo/fluent_assertions.svg) was created by
-
-* [IUsername](https://github.com/IUsername)
-
-If you have any comments or suggestions, please let us know via [twitter](https://twitter.com/search?q=fluentassertions&src=typd), through the [issues](https://github.com/dennisdoomen/FluentAssertions/issues) page, or through [StackOverflow](http://stackoverflow.com/questions/tagged/fluent-assertions).
+My name is [Dennis Doomen](https://twitter.com/ddoomen) and I work for [Aviva Solutions](https://www.avivasolutions.nl/) in The Netherlands. I maintain a [blog](https://www.continuousimprover.com/) on my everlasting quest for knowledge that significantly improves the way you build your key systems in an agile world. Fluent Assertions is one of those aspects of that. Since a year two, [Jonas Nyrup](https://github.com/jnyrup) has joined to project and help bring it forward. 
```

Comment:
"Since a year two" -> "Since 2018"
(Had to look up our Slack conversations to see when I formally joined)